packages/graphql/src/translate/translate-read.ts

```diff
@@ -163,16 +163,6 @@ export function translateRead(
             matchNode,
         ]);
 
-        if (!projectionSubqueries.empty && hasOrdering) {
-            addSortAndLimitOptionsToClause({
-                optionsInput,
-                target: matchNode,
-                projectionClause: returnClause,
-                nodeField: node.singular,
-                fulltextScoreVariable: context.fulltextIndex?.scoreVariable,
-            });
-        }
-
```

Comment:
This code is not exercised as it is a normal sort behaviour inside a top-level connection